### PR TITLE
SC-1527

### PR DESCRIPTION
**Pull request by anmol2302**
**Ticket Ref:** [SC-1527](https://project-sunbird.atlassian.net/browse/SC-1527)
**Release:** 2.5.0
1: added certificate 
            

-  `certreg/v1/certs/add` API
- `certreg/v1/certs/validate` API
- `certreg/v1/certs/download` API
- `certreg/v1/certs/verify` API

All the API's above are of Certificate_registry service.

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://540.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br /> - http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

File: apis/certificateregistry/certificateregistryapi.yaml [hunk 1]
```diff
@@ -0,0 +1,376 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: Certificate api
+  description: >-
+    This api is used for store download validate the course certificate.
```
Comment (manzarul) [inline apis/certificateregistry/certificateregistryapi.yaml]: Certificate registry is used for storing , downloading and validation of certificate.

File: apis/certificateregistry/certificateregistryapi.yaml [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+          type: string
+          description: 'A unique ID that identifies the request in case the same API is executed multiple times.'
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'OK ! Operation successful."Certificate generation" operation was executed successfully.'
```
Comment (manzarul) [inline apis/certificateregistry/certificateregistryapi.yaml]: This is for validate , so change description.

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://540.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br /> - http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

File: apis/certificateregistry/certificateregistryapi.yaml (hunk 1 above)
Comment (anmol2302) [reply apis/certificateregistry/certificateregistryapi.yaml]: done

File: apis/certificateregistry/certificateregistryapi.yaml (hunk 2 above)
Comment (anmol2302) [reply apis/certificateregistry/certificateregistryapi.yaml]: done

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://540.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br /> - http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://540.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br /> - http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://540.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br /> - http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://540.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br /> - http://540.qa.docs.sunbird.org/2.5.0/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**